src/legacy/core_plugins/kibana/public/discover/components/field_chooser/_field_chooser.scss

```diff
@@ -22,5 +22,8 @@
 }
 
 .dscToggleFieldFilterButton {
-  min-height: $euiSizeXL;
+  width: calc(100% - 8px);
+  color: #006BB4;
+  padding-left: 2px;
+  margin-left: 4px;
```

Comment:
These should EUI size variables such as `$euiSizeS`, `$euiSizeXS`, etc.

You can find a list of available SASS variables here: https://elastic.github.io/eui/#/guidelines/sass

There is no 2px var, so you can either remove the `padding-left` or use the XS value.

Reply:
Same for the color value... the EUI equivalent would be `$euiColorPrimary`